spinta/utils/data.py

```diff
@@ -71,6 +74,8 @@ def take(keys, *args):
     for k in keys:
         for v in args:
             for x in k:
+                if v and not isinstance(v, Iterable):
+                    raise InvalidNestedValue(value=v)
```

Comment:
We should not use higher level components in a lower level function. Exceptions from spinta.exceptions are high level exceptions and spinta.utils is a set of low level functions.

Since this is probably related to case when `ref` type receives a scalar value, then this kind of check should be done in a `simple_data_check` command.

See the command flow below:

![image](https://user-images.githubusercontent.com/297583/225121773-96e30f61-9c7f-43a4-b6dc-82f0432872e8.png)